guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -52,8 +62,21 @@ On the main {Project} server:
                        --certs-tar "~/myproxy.example.com-certs.tar"
 ----
 +
-For more information on custom SSL certificates signed by a Certificate Authority, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
+
+. If you use custom SSL certificates on the {smart-proxy-context} server, use the following command to regenerate the certificate tar file on the {Project} server:
```

Comment:
```suggestion
. If you use custom SSL certificates on your {smart-proxy-context} server, use the following command to regenerate the certificate tar file on {ProjectServer}:
```

Can we use "SmartProxyServer" instead of "{smart-proxy-context} server"?

Reply:
Will do.